tutorial3/step5.md

```diff
@@ -49,17 +49,11 @@ Sort entries alphabetically if none of -cftuvSUX nor --sort is specified.
 
 It is a shorter version of the man page and it just prints it to screen which 
 can be very convenient.  Help text can be pretty comprehensive like it is here 
-but it can also be just usage instructions so just something like this:
+but it can just contain simple usage instructions.
 
-```
-Usage: ls [OPTION]... [FILE]...
-List information about the FILEs (the current directory by default).
-```
-
-Almost all software installed on your machine will have a `man page` on a 
-unix or unix-like system.  However, if there is no manual for a command line 
-tool adding `--help` to the command is often a good idea as it can  bring up 
-help info.
+Almost all software installed on your machine will have a `man page`.  However,
```

Comment:
I'm never sure whether to say your machine or unix like system as I am concerned windows users will get confused if they try this themselves. This is the case for all corrections that brush upon this including the one about absolute paths beginning with a /

Reply:
Mm, I see your point. I don't like talking about "UNIX" as it's complex jargon that doesn't have any bearing on the tutorial content. Perhaps we should explicitly state somewhere that this won't work on Windows? Or something about compatibility; where this knowledge can and can't be used.

Good point about the absolute paths too. It's worth noting that in Git Bash (on Windows), the paths are converted from `C:\Users\Edward` to `/c/Users/Edward`.